src/realm/mixed.cpp

```diff
@@ -30,6 +30,8 @@ namespace realm {
 namespace _impl {
 
 static const int sorting_rank[19] = {
+    // OBS. Changing these values breaks the file format for Set<Mixed>
```

Comment:
OBS doesn't translate for an english reader 😄

Reply:
Changed words